### PR TITLE
Add a native tokens tutorial and exercises to a newly created MA folder

### PR DESCRIPTION
**Pull request by olgahryniuk**
@Jimbo4350 As per our discussion yesterday, I've created a new 'native tokens' folder where I moved 'multi-assets', added a tutorial, exercises, and exchange management scenarios. Please review & thank you!
The existing multi-assets file is different from exchange-scenarios so there was no need in merging them together.

### CODE REVIEW

**Comment by Jimbo4350** [comment]
@olgahryniuk you need to create a branch off of `master`. Currently you are committing directly to `master`.

**Comment by olgahryniuk** [comment]
Jordan, this is a separate branch. I forked the node repo and renamed it to native tokens: https://github.com/input-output-hk/cardano-node-native-tokens-folder-TW.

File: doc/reference/native-tokens/02-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+- create new currencies and assets;
+- submit and send transactions containing multi-asset tokens;
+- send and receive token bundles;
+- manage your addresses and values.
+
+> Note: Users who do not need to create new assets (“token holders”) will be able to send and receive existing multi-asset tokens using a wallet such as Daedalus or Yoroi, and with no requirement to use any CLI commands.
+
+To start, please ensure that you are familiar with setting up and operating the [Cardano node](https://github.com/input-output-hk/cardano-node). Alternatively, see instructions on how to [start your node](https://github.com/input-output-hk/cardano-node/blob/master/doc/stake-pool-operations/2_start_your_nodes.md) to submit the commands. You will not need to set up and start a full block producing node (“stake pool”), just a much simpler relay node. This node will need to connect to a Cardano network that is capable of processing native tokens (e.g., the native token pre-production environment (PPE), or the Cardano Mainnet once it has been upgraded).
+
+### Configuring cardano-node to run using cardano-cli
```
Comment (Jimbo4350) [inline doc/reference/native-tokens/02-getting-started.md:18]: A general version of this section should live in the `README.md` in the `cardano-node` dir and then referenced here.

File: doc/reference/native-tokens/02-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,582 @@
...
+- `${assetName}` (where assetName is hex-encoded 61737365744e616d65)
+
+Values for other assets can be specified as:
+
+- `${quantity} ${policyId}.${assetName}`
+- `${quantity} ${policyId}` 
+
+Where `quantity` is a signed integer and `policyId` is a hex-encoded policy ID [a script hash]), and `assetName` is a hex-encoded assetName.
+
+#### Syntax of multi-asset values
```
Comment (Jimbo4350) [inline doc/reference/native-tokens/02-getting-started.md:74]: A lot of this document covers things spoken about in `doc/reference/multi-assets.md`. Please compare both and consolidate the two into a single document.

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]

File: doc/reference/native-tokens/02-getting-started.md (hunk 1 above)
Comment (olgahryniuk) [reply doc/reference/native-tokens/02-getting-started.md:1]: This is pushed. Closing now

File: doc/reference/native-tokens/02-getting-started.md (hunk 2 above)
Comment (olgahryniuk) [reply doc/reference/native-tokens/02-getting-started.md:57]: Done

File: doc/reference/native-tokens/02-getting-started.md [hunk 3]
```diff
@@ -0,0 +1,565 @@
...
+```
+
+The response should show no funds:
+
+```bash
+                           TxHash                                 TxIx        Amount
+--------------------------------------------------------------------------------------
+```
+
+6. Fund the address and check again:
```
Comment (Jimbo4350) [inline doc/reference/native-tokens/02-getting-started.md:206]: We should point to documentation describing how to fund your testnet address.

File: doc/reference/native-tokens/02-getting-started.md [hunk 4]
```diff
@@ -0,0 +1,565 @@
...
+
+#### Build the raw transaction
+
+1. Use this code to build the raw transaction:
+
+```
+$ echo -n "melcoin" | xxd -ps 
+6d656c636f696e
+
+./cardano-cli transaction build-raw \
```
Comment (Jimbo4350) [inline doc/reference/native-tokens/02-getting-started.md:268]: Is there a reason we are using `build-raw` and not `build` in the documentation?

**Review by Jimbo4350 — CHANGES_REQUESTED** [review]
Nice, good work. A few more comments to discuss.

File: doc/reference/native-tokens/02-getting-started.md [hunk 5]
```diff
@@ -0,0 +1,540 @@
+Before you start, you should:
+
+- have the [latest version of the node](https://github.com/input-output-hk/cardano-node/releases)
```
Comment (Jimbo4350) [inline doc/reference/native-tokens/02-getting-started.md:3]: :+1:

File: README.rst [hunk 6]
```diff
@@ -405,15 +405,19 @@ Byron vote submission:
...
+To help you get started, see:
 
-`Cardano Forum discussion <https://forum.cardano.org/c/developers/cardano-tokens/150>`_
+- `Cardano Forum discussion <https://forum.cardano.org/c/developers/cardano-tokens/150>`_
 
-`Documentation for native tokens <https://docs.cardano.org/native-tokens/learn>`_
+- `Ledger explanations: native tokens <https://cardano-ledger.readthedocs.io/en/latest/>`_. Covers explainers about assets, tokens, token bundles, minting policies, comparison to ERC20, and minimum ada value requirements.
 
-You can also read more about `native tokens and how they compare to ada and ERC20 <https://github.com/input-output-hk/cardano-ledger-specs/blob/master/doc/explanations/features.rst>`_. Browse native tokens created on the Cardano blockchain and see their transactions in an interactive dashboard that allows filtering and searching: nativetokens.da.iogservices.io.
+- A tutorial on how to get started with native tokens (LINK). Explains how to create new currencies and assets, submit and send transactions containing multi-asset tokens, send and receive token bundles, manage your addresses and values.
```
Comment (Jimbo4350) [inline README.rst:416]: I think you forgot to link to the native tokens exercises here

File: doc/reference/native-tokens/01-multi-assets.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
++ *PolicyID* - the unique identifier that is associated with a minting policy (hash of the minting policy).
++ *Asset name* - an (immutable) property of an asset that is used to distinguish different assets within the same policy. Unlike the policyID, the asset name does not refer to any code or set of rules. It is an arbitrary sequence of bytes. In simplest case it is an ASCII-encoded common word, eg. ‘couttscoin’. In this document we use asset name `"636f75747473636f696e" = hex("couttscoin")` as an example.
+
+Tokens that have the same asset ID have the property of being fungible with each other, and are not fungible with tokens that have a different asset ID.
+
+Further reading:
+
+- [Native token explainers](https://cardano-ledger.readthedocs.io/en/latest/)
+- Getting started with native tokens(LINK). Includes script examples for minting, transferring, and burning tokens.
```
Comment (Jimbo4350) [inline doc/reference/native-tokens/01-multi-assets.md:19]: And here as well: (LINK)

**Review by Jimbo4350 — COMMENTED** [review]
Looking good! Three minor changes.

**Review by Jimbo4350 — APPROVED** [review]

**Pull request merged**